polaris.shopify.com/scripts/gen-assets.mjs

```diff
@@ -2,9 +2,6 @@ import genSiteMap from './gen-site-map.mjs';
 import genCacheJson from './gen-cache-json.mjs';
 import genOgImages from './gen-og-images.mjs';
 
-const genAssets = async () => {
-  await Promise.all([genSiteMap(), genCacheJson()]);
-  await genOgImages();
-};
-
-await genAssets();
+await genSiteMap();
+await genCacheJson();
+await genOgImages();
```

Comment:
We support Node 14+ which in turn supports top level await. No need to wrap this in a function.

Since the `genCacheJson` command is extremely fast < 1s, and the `genOgimages` is extremely slow > 30s, it didn't make sense to bother running them in parallel. This frees us up to have each script display their own spinner without colliding.